concepts/basics/introduction.md

```diff
@@ -0,0 +1,92 @@
+# Haskell Syntax Basics
```

Comment:
I wondered why the introduction and about are so similar, but I see this is the case for https://github.com/exercism/fsharp/tree/main/concepts/basics as well. So I won't ask more questions about why they're the same, but for every change that is made to about.md please do remember to make it to introduction.md as well.